adapters/oidc/js/src/main/resources/keycloak.js

```diff
@@ -1554,3 +1625,25 @@
         }
     }
 })( window );
+
+/** Additional support libraries */
+
+/**
+ * [js-sha256]{@link https://github.com/emn178/js-sha256}
```

Comment:
@thomasdarimont any reason to prefer a third party library over `crypto.subtle.digest`?

Reply:
That's what I tried first, but when I contributed the feature `crypto.subtle.digest` function wasn't available in the major browsers. Perhaps the situation changed in the meantime...

It's a similar story for the the base64-js library, which I needed because the `btoa` base64 encodings cannot deal with Uint8ArrayBuffer's and unicode.

See my original commit:
https://github.com/thomasdarimont/keycloak/commit/f05066d430f6504246f7e518a124aef2ef5195b8

Reply:
Of course, it would be better if this would be possible without embedding additional libraries, but I didn't manage to get the base64 encoding or sha256 digest working, without implementing the algorithms myself.